test/integration/sanity/multi-case-url.test.js

```diff
@@ -0,0 +1,117 @@
+var expect = require('chai').expect;
+describe('Different case url', function() {
+
+    describe('Lowercase', function() {
+        var testrun;
```

Comment:
Variable `testrun` can be declared in the parent `describe` block instead of declaring in every child blocks.